### DISCUSSION TITLE
Hi, I need some help with the scriptscheduler

### DISCUSSION THREAD

**Discussion by Hildero** [ORelio/Minecraft-Console-Client#2780] [Q&A]
I dont understand why the scriptscheduler doesnt recognise the command. This is what I put down in minecraftclient, and made a myscripts.txt file with commands. but it just wont run the script through the scriptscheduler. Any hints please?
![image](https://github.com/user-attachments/assets/e164ef50-c5c0-4dc6-8ed3-b7892496be72)

**Comment by breadbyte** [comment]
Remove the `/` in `/script`.
